jbpm-console-ng-business-domain/jbpm-console-ng-business-domain-api/src/main/java/org/jbpm/console/ng/bd/model/TaskDefSummary.java

```diff
@@ -24,14 +24,18 @@
     private long id;
     private String name;
 
-    public TaskDefSummary(long id, String name) {
-        super();
+    public TaskDefSummary(long id, final String name) {
+        this(name);
         this.id = id;
-        this.name = name;
+    }
 
+    public TaskDefSummary(final String name) {
+        super();
+        this.name = name;
     }
 
     public TaskDefSummary() {
+        super();
```

Comment:
Explicit call to super(); is not necessary, because call the accessible no-args constructor from the superclass is automatically added by compiler.

The same comment applies to
NodeInstanceSummary
ProcessInstanceSummary
ProcessInstanceVariableSummary
TaskDefSummary
TaskAssignmentSummary

but I guess it doesn't hurt.